{"name": "test_signin_api_using_invalid_special_and_integerValue_phone", "status": "failed", "statusDetails": {"message": "AssertionError: assert 200 == 400\n +  where 200 = <Response [200]>.status_code", "trace": "self = <Server.API.Test.test_signup_trado_commerce_api.TestLogin_API object at 0x00000148E5977F70>\n\n    @allure.description('test_signin_api_using_invalid_special_and_integerValue_phone')\n    def test_signin_api_using_invalid_special_and_integerValue_phone(self):\n        url = SignupConstants.url\n        data = SignupConstants.invalid_special_and_integerValue_phone\n        res = requests.post(url, data)\n        res_data = res.json()\n>       assert res.status_code == 400\nE       assert 200 == 400\nE        +  where 200 = <Response [200]>.status_code\n\ntest_signup_trado_commerce_api.py:90: AssertionError"}, "description": "test_signin_api_using_invalid_special_and_integerValue_phone", "start": 1677093976073, "stop": 1677093976580, "uuid": "97c25f28-96bb-44b8-9bf4-3861ad7a0c8b", "historyId": "98cf8d1021d7494c0ae00099dca8ea6e", "testCaseId": "cf03b478855db4f99d53a2e4c0e96ecb", "fullName": "test_signup_trado_commerce_api.TestLogin_API#test_signin_api_using_invalid_special_and_integerValue_phone", "labels": [{"name": "suite", "value": "test_signup_trado_commerce_api"}, {"name": "subSuite", "value": "TestLogin_API"}, {"name": "host", "value": "LAPTOP-CFNBSI63"}, {"name": "thread", "value": "20924-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_signup_trado_commerce_api"}]}